privacy_evaluator/attacks/property_inference_attack.py

```diff
@@ -292,21 +303,19 @@ def prediction_on_specific_property(
         feature_extraction_target_model: np.ndarray,
         shadow_classifiers_neg_property: list,
         ratio: float,
-        size_set: int,
     ) -> np.ndarray:
         """
         Perform prediction for a subattack (specific property)
         :param feature_extraction_target_model: extracted features of target model
         :param shadow_classifiers_neg_property: balanced shadow classifiers negation property
         :param ratio: distribution for the property
-        :param size_set: size of one class from data set
         :return: Prediction of meta-classifier for property and negation property
         """
 
         # property of given ratio, only on class 0 and 1 at the moment
         property_num_elements_per_classes = {
-            0: int((1 - ratio) * size_set),
-            1: int(ratio * size_set),
+            0: int((1 - ratio) * self.size_set),
```

Comment:
Here we use the self.classes to select the classes instead of 0 and 1 beeing hard-coded

Reply:
Changed in new pull request so I resolved the conversation.